docs/clusters/blanca/blanca.md

```diff
@@ -75,7 +75,7 @@ sinfo --format="%N | %f" --partition="blanca-curc"
 
 ### Requesting GPUs in jobs
 
-Using GPUs in jobs requires one to use the General Resource ("Gres") functionality of Slurm to request the gpu(s).  At a minimum, one would specify `#SBATCH --gres=gpu` in their job script to specify that they would like to use a single gpu of any type.  One can also request multiple GPUs on nodes that have more than one, and a specific type of GPU (e.g., V100, A100) if desired.  The available Blanca GPU resources and configurations can be viewed as follows on a login node with the `slurm/blanca` module loaded:
+Using GPUs in jobs requires one to use the General Resource ("Gres") functionality of Slurm to request the GPU(s).  At a minimum, one would specify `#SBATCH --gres=gpu` in their job script to specify that they would like to use a single GPU of any type.  One can also request multiple GPUs on nodes that have more than one, and a specific type of GPU (e.g., V100, A100) if desired.  The available Blanca GPU resources and configurations can be viewed as follows on a login node with the `slurm/blanca` module loaded:
```

Comment:
Can you go through this page and make places where there are multiple examples into tabs?